css/css-animations/META.yml

```diff
@@ -1,11 +1,21 @@
 links:
 - product: firefox
+  url: https://bugzilla.mozilla.org/show_bug.cgi?id=1634352
   results:
   - test: animation-important-001.html
-  url: https://bugzilla.mozilla.org/show_bug.cgi?id=1634352
 - product: firefox
+  url: https://bugzilla.mozilla.org/show_bug.cgi?id=1635063
   results:
-  - subtest: CSS Animations targetting (pseudo-)elements should have correct order
+  - test: Document-getAnimations.tentative.html
+    subtest: CSS Animations targetting (pseudo-)elements should have correct order
       after sorting (::marker)
-    test: Document-getAnimations.tentative.html
-  url: https://bugzilla.mozilla.org/show_bug.cgi?id=1635063
\ No newline at end of file
+- product: chrome
+  url: crbug.com/1064065
```

Comment:
This is bug created by Flake Portal but doesn't seem to flaky on [wpt](https://wpt.fyi/results/css?label=master&label=experimental&max-count=10&product=chrome&q=seq%28%28status%3APASS%7Cstatus%3AOK%29%20%28status%3A%21PASS%26status%3A%21OK%26status%3A%21unknown%29%29%20seq%28%28status%3A%21PASS%26status%3A%21OK%26status%3A%21unknown%29%20%28status%3APASS%7Cstatus%3AOK%29%29)